spec/features/admin/products_spec.rb

```diff
@@ -114,6 +114,82 @@
       expect(page).to have_content "Unit value can't be blank"
     end
   end
+  
+  describe "soft-deleting" do
```

Comment:
I would keep this add the abstraction level of features so this doesn't get out of date if we no longer implement it with soft-deletion.

Reply:
Removed the `soft-` part of it.